src/backend/api/meals.js

```diff
@@ -1,14 +1,70 @@
+const knex = require("knex");
 const express = require("express");
-const router = express.Router();
-const knex = require("../database");
```

Comment:
You should have kept this line as is. The `knex` this imports is not the `knex` library, but your connection to your database, that was created in the `database.js` file. Otherwise, this code does not work.|

You replaced it with `const knex = require("knex");` which just imports the `knex` library, and NOT a database connection.

Reply:
changes made already thanks @kiko-bkr